torch/_dynamo/convert_frame.py

```diff
@@ -873,17 +870,11 @@ def count_args(code: CodeType) -> int:
 
         return guarded_code
 
-    chromium_event_log = get_chromium_event_logger()
-
-    chromium_event_log.reset()
-    chromium_start_time = time.time_ns()
-    chromium_event_log.log_event_start(
-        "dynamo", chromium_start_time, {}, log_pt2_compile_event=True
-    )
-
     metrics_context = get_metrics_context()
```

Comment:
@xmfan ; I guess there will be a merge conflict around here with your PR. You wanna land first?

Reply:
should I still use chromium_event_log directly here? I don't need a context manager since my start and end times are pre-recorded

Reply:
@xmfan yeah I think it's fine to just create your separate chromium event directly. I introduced the contextmanager here to help with the sequencing of the "dynamo" chromium event and the dynamo_compile scuba log (I need the scuba logging to complete first, then the chromium event).

Reply:
I was hoping to let you in line first and I'd handle the merge, but I'm out for the week so I just landed. Sounds like Ed is voting for doing the compiled autograd logging earlier and potentially changing the compile_id format so maybe your PR moves the logging to another location anyway?